packages/editor/src/AppModel/ComponentModel.ts

```diff
@@ -183,7 +183,7 @@ export class ComponentModel implements IComponentModel {
       parent.children[slot] = [];
     }
 
-    parent.children[slot].push(this);
+    parent.children[slot].unshift(this);
```

Comment:
Why use unshift? `append` usually means add to the last position.
If you want add a component before other siblings, you should first call `appendTo`, then call `moveAfter`.

Reply:
I want to append it to the front of the slot when dropping a component into the slot item. I split the `appendTo` into `appendToRoot` and `appendToSlotFront` two functions. What do you think of this approach?

Reply:
Maybe it is better to add component at the first in dropping situation, but in other situation it is not. For example, user drop component into component element in canvas. Actually, we cannot prediect whether user wants to append component to the first or the last.
There is no way to satisfy all the situations. So I think we just keep it simple and make no changes.

Reply:
Done. I revert the related codes.